app/javascript/chat/actions/actions.js

```diff
@@ -56,20 +57,30 @@ export function editMessage(editedMessage, successCb, failureCb) {
     .catch(failureCb);
 }
 
-export function sendOpen(activeChannelId, successCb, failureCb) {
-  fetch(`/chat_channels/${activeChannelId}/open`, {
+// export function sendOpen(activeChannelId, successCb, failureCb) {
+//   fetch(`/chat_channels/${activeChannelId}/open`, {
+//     method: 'POST',
+//     headers: {
+//       Accept: 'application/json',
+//       'X-CSRF-Token': window.csrfToken,
+//       'Content-Type': 'application/json',
+//     },
+//     body: JSON.stringify({}),
+//     credentials: 'same-origin',
+//   })
+//     .then((response) => response.json())
+//     .then(successCb)
+//     .catch(failureCb);
+// }
+
+export async function sendOpen(activeChannelId) {
+  const response = await request(`/chat_channels/${activeChannelId}/open`, {
     method: 'POST',
-    headers: {
-      Accept: 'application/json',
-      'X-CSRF-Token': window.csrfToken,
-      'Content-Type': 'application/json',
-    },
-    body: JSON.stringify({}),
     credentials: 'same-origin',
-  })
-    .then((response) => response.json())
-    .then(successCb)
-    .catch(failureCb);
+    body: JSON.stringify({}),
```

Comment:
The `request` utility function converts the body automatically to JSON. See https://github.com/forem/forem/blob/master/app/javascript/utilities/http/request.js#L35

```suggestion
    body: {},
```